docs/resources/Emoji.md

```diff
@@ -11,7 +11,7 @@
 | --------------- | ---------------------------------------------------------------- | ------------------------------------------------------------------------- |
 | id              | ?snowflake                                                       | [emoji id](#DOCS_REFERENCE/image-formatting)                              |
 | name            | ?string (can be null only in reaction emoji objects)             | emoji name                                                                |
-| roles?          | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids | roles this emoji is whitelisted to                                        |
+| roles?          | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids | roles allowed to use this emoji                                        |
```

Comment:
This changes the semantics of the statement. Poor grammar aside, "roles this emoji is whitelisted to" makes sense while "roles allowed to use this emoji" does not because roles don't use emojis; members/users do.

Reply:
how about ``if not empty, only members with at least one of these roles can use this emoji`` ?

Reply:
Seems okay. It is explicit.

Reply:
@Fyko +1 to @AEnterprise's wording for these role entries.